experiments/experiments-impl/src/main/java/com/duckduckgo/experiments/impl/store/VariantManagerEntity.kt

```diff
@@ -26,26 +26,10 @@ import com.squareup.moshi.Types
 @Entity(tableName = "experiment_variants")
 data class ExperimentVariantEntity(
     @PrimaryKey val key: String,
-    val weight: Float?,
-    val filters: VariantFiltersEntity?,
+    val weight: Double?,
+    val localeFilter: List<String> = emptyList(),
```

Comment:
Do we need local filters?

Reply:
SERP filters are local ("sc" and "se" variants), and we only support locale filters remotely just now (per tech design specifications). In the future we can add more filters columns if we decide to support them.